README.md

```diff
@@ -1,9 +1,23 @@
-# Sass Modules Migrator
+Easily upgrade your Sass stylesheets!
 
-This tool is designed to help migrate stylesheets from the old `@import`
-functionality to [the new module system][module-spec].
+<table>
+  <tr>
+    <td>
+      <img width="118px" alt="Sass logo" src="https://rawgit.com/sass/sass-site/master/source/assets/img/logos/logo.svg" />
+    </td>
+    <td valign="middle">
+      <a href="https://www.npmjs.com/package/sass-migrator"><img width="100%" alt="npm statistics" src="https://nodei.co/npm/sass-migrator.png?downloads=true"></a>
+    </td>
+    <td valign="middle">
+      <a href="https://travis-ci.org/sass/sass-migrator"><img alt="Travis build status" src="https://api.travis-ci.org/sass/sass-migrator.svg?branch=master"></a>
+    </td>
+  </tr>
+</table>
 
-This is still a work-in-progress, but should eventually implement everything
-described in [sass/sass#2575](https://github.com/sass/sass/issues/2575).
+The Sass migrator automatically updates stylesheets that deprecated behavior to
```

Comment:
Should this say "that contain deprecated behavior"?

Reply:
Done.